src/components/breadcrumb/Breadcrumb.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import * as React from 'react';
-import { injectIntl, type IntlShape } from 'react-intl';
```

Comment:
Can we retain the unified import and use it in the other affected modules?

Reply:
Does flow treat those imports the same way TS does?

Reply:
@greg-in-a-box I believe Flow doesn't directly support the 'import type' syntax. However, I continue to use it in the same way as before the upgrade, ('import type { InjectIntlProvidedProps } from 'react-intl'). No errors have been reported, due to settings in .vscode.